web/styles/settings.css

```diff
@@ -14,6 +14,7 @@ label {
 label,
 h4,
 h3,
+div,
```

Comment:
Does this add new CSS to all links inside divs in the app, aka just about every link? I don't think we want to do this.

My guess is there's a better way to do this, but also please take some time to read about thoughtful CSS scoping -- it's an important topic and it shouldn't be reviewers' job to catch this sort of issue in your work.

Reply:
Sorry for the inconvenience. I was experimenting with different things and forgot to push the correct logic for this fix. I will review the changes more carefully next time.